content/docs/tasks/policy-enforcement/control-headers/index.md

```diff
@@ -23,7 +23,7 @@ This task demonstrates how to use a policy adapter to manipulate request headers
   configuration for the `httpbin` service containing two route rules that allow traffic for paths `/headers` and
   `/status`:
 
-    {{< text yaml >}}
+    {{< text bash >}}
```

Comment:
Actually, this should be "text bash yaml" indicating a command line, followed by yaml output.